### PR TITLE
Added on the fly reaction map editing, corrected typo, and custom rpc

### PR DESCRIPTION
**Pull request by PlerionPlorion**
edit the reaction map using commands available only to users with "Manage Roles"

prefix: p?
usage: p?updatemap [keyword] [emoji(s)]
p?delete [keyword]

(discord literally has no way to get emojis from slash commands I wasted an hour on that)

### CODE REVIEW

File: src/events/ready.js [hunk 1]
```diff
@@ -1,12 +1,52 @@
...
+                        type: 0, // enum: 0 = playing, 1 = streaming, 2 = listening, 3 = watching, 4 = custom, 5 = competing
+                        status: "online",
+                        afk: false,
+                    },
+                ],
+            });
+        } else {
+            console.log("No roles found.");
+        }
+        // console.log("buh");
```
Comment (GalexY727) [inline src/events/ready.js:50]: 🐐

**Review by GalexY727 — COMMENTED** [review]
This is good but I'm going to extract it to its own file so that it changes spontaneously

**Comment by GalexY727** [comment]
This just came to mind, but we would need to solve the source control side of things for editing the reaction map considering that this is running on a server that is currently just running `git pull; node ./src/index.js` on startup. If we edited the reaction map and restarted the bot it wouldn't pull since we have local changes on our branch. I don't want to run `git fetch origin; git reset --hard origin/main` because that would override the local changes, and I don't want to run a rebase pull since that would create a staggered new commit which would branch from and desync with GitHub.

**Comment by PlerionPlorion** [comment]
buh, ig you need the server to autocommit file changes.

**Comment by GalexY727** [comment]
Which would need to be integrated into the Js...

**Comment by PlerionPlorion** [comment]
alright I'll see if I can figure that out.

**Comment by GalexY727** [comment]
I think you can do it with some node module but I don't want to add bloaty deps

**Comment by PlerionPlorion** [comment]
child_process

**Comment by PlerionPlorion** [comment]
ok it works now

**Review by GalexY727 — APPROVED** [review]
looks nice :)

**Pull request merged**